packages/main/src/TableRow.js

```diff
@@ -259,6 +289,44 @@ class TableRow extends UI5Element {
 		this.fireEvent("selection-requested", { row: this });
 	}
 
+	/**
+	 * Toggles focus between the table row's root and the last focused nested element.
+	 * @private
+	 * @param {Object} activeElement The currently focused element
+	 */
+	_handleF7(activeElement) {
+		const elements = this._tabbableElements;
+
+		if (!elements.length) {
+			return;
+		}
+
+		const table = this.parentElement;
```

Comment:
Again, you should not access the table but instead fire an event telling it to do something.

Reply:
Implemented as part of #4698.